### ISSUE TITLE
Mute (skip) unnecessary go straight that might cause confusion

### ISSUE THREAD

**Issue by DmitryAlexei** [osmandapp/OsmAnd#20262]
### Description

In some situations, the directional arrow in the Lanes widget doesn't match the Next turn widget

### Steps to reproduce

Coordinates of the issue https://www.openstreetmap.org/#map=17/48.53686/8.94161

### Actual result

![image](https://github.com/osmandapp/OsmAnd/assets/116079541/1a6b4111-8bc2-4509-b8e4-e10214be49e1)

### Expected result

When the Next turn widget and Lanes widget show info about the same turn, the directional arrows in both widgets should be the same

### Your Environment (required)

**WARNING** Crash-Logs **MAY** contain information you deem sensitive.
Review this **CAREFULLY** before posting your issue!

```
OsmAnd Version:
Android/iOS version:
Device model:
Crash-Logs: ?
```

**Comment by RZR-UA** [comment]
Incorrect Turn-right (TL|+TR) ?

https://github.com/osmandapp/OsmAnd/assets/90444451/4fef7ac5-62fc-42e9-a488-606962c1f707

**Comment by RZR-UA** [comment]
Web: Go ahead (TL|+TR,C) onto Poltringer Hauptstraße L 359

Correct Go ahead but incorrect TL|+TR,C ?

[link](https://test.osmand.net/map/navigate/?start=48.535466,8.947334&end=48.536915,8.946808&profile=rescuetrack&params=rescuetrack,routing:hh_cpp,approximation:geo_cpp#19/48.53615/8.94695)

![web](https://github.com/osmandapp/OsmAnd/assets/90444451/d5677a36-977c-487c-a977-55dd9f0e845c)

**Comment by RZR-UA** [comment]
OSM Node: https://www.openstreetmap.org/node/291245963

**Comment by RZR-UA** [comment]
- [ ] mute subject
- [ ] pass turn-lanes tests
- [ ] add new test for this case

**Comment by DmitryAlexei** [comment]
https://osmand.net/map/navigate/?start=48.535043,8.948107&end=48.536819,8.946701&profile=car&pin=54.820779,26.834310#18/48.53591/8.94786
<table style="width:100%">
  <tr>
<th><img src="https://github.com/user-attachments/assets/d1edd0d3-09b3-43b3-b975-176baead650f"  width="800" height="auto"/> </td>
  </tr>
</table>

OsmAnd 4.8.3, released: 2024-07-18 TestFlight 4.8.0.1 
<table style="width:100%">
  <tr>
<th><img src="https://github.com/user-attachments/assets/a86f7779-0ab0-4fe8-82ac-3addd86c2def"  width="300" height="auto"/> </th>
 <th><img src="https://github.com/user-attachments/assets/1ac27046-fbf3-4879-a809-c4090063e88d"  width="700" height="auto"/> </td>
  </tr>
</table>
